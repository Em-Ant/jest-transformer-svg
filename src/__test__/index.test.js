import React from 'react';
import { render } from '@testing-library/react';

import SvgComponent from './example-asset.svg';

function TestComponent() {
  return (
    <div>
      <span className="heading">Title</span>
      <SvgComponent foo="bar" style={{ height: '100px' }} />
    </div>
  );
}

test('TestComponent should render svg child with correct props', () => {
  const { container } = render(<TestComponent />);
  expect(container).toMatchSnapshot();
});
