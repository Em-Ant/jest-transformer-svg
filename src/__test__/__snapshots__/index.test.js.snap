// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent should render svg child with correct props 1`] = `
<div>
  <div>
    <span
      class="heading"
    >
      Title
    </span>
    <svg
      data-file-name="SvgExampleAsset"
      foo="bar"
      style="height: 100px;"
    />
  </div>
</div>
`;
